### PR TITLE
add stkXPRT to Persistence

### PR DESCRIPTION
**Pull request by saurabhfegade**

### CODE REVIEW

File: persistence/assetlist.json [hunk 1]
```diff
@@ -645,6 +645,37 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/huahua.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/huahua.svg"
       }
+    },
+    {
+      "description": "PSTAKE Liquid-Staked XPRT",
```
Comment (JeremyParish69) [inline persistence/assetlist.json:650]: For liquid staking tokens, please define the `traces`

Comment (saurabhfegade) [reply persistence/assetlist.json:650]: @JeremyParish69 done please check

File: persistence/assetlist.json [hunk 2]
```diff
@@ -645,6 +645,47 @@
...
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "persistence",
+            "base_denom": "uxprt"
+          },
+          "provider": "pSTAKE"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkxprt.png",
```
Comment (JeremyParish69) [inline persistence/assetlist.json:680]: Could I ask for a better png file? It looks blurry, has a shadow, and has extra padding. In contrast, the svg looks great. We aim to have the images within the same object have to exact same appearance.

Comment (saurabhfegade) [reply persistence/assetlist.json:680]: Sure. Updated it. Can you please check if the updated image works? @JeremyParish69

**Pull request merged**